packages/components/src/components/action-pad/action-pad.tsx

```diff
@@ -23,7 +23,7 @@ declare global {
 }
 
 /**
- * @deprecated Use the `calcite-action-bar` component instead.
+ * @deprecated in v5.0, removal target v6.0 - Use the `calcite-action-bar` component instead.
```

Comment:
For consistency should we list the full releases across to include the patch numbers, or alternatively, should the patch numbers be excluded across the update?
- Patch numbers: `6.0.0`
- Minor numbers: `6.0`

```suggestion
 * @deprecated in v5.0.0, removal target v6.0.0 - Use the `calcite-action-bar` component instead.
```